### PR TITLE
Ballot#187

### PR DESCRIPTION
**Pull request by Healthedata1**
Ballot#187 #219

### CODE REVIEW

File: spec/scopes-and-launch-context/index.md [hunk 1]
```diff
@@ -7,9 +7,14 @@ SMART on FHIR's authorization scheme uses OAuth scopes to communicate (and
 negotiate) access requirements. Providing apps with access to broad data sets is consistent with current common practices (e.g. interface engines also provide access to broad data sets); access is also limited based on the privileges of the user in context.  In general, we use scopes for three kinds of data:
 
 1. Clinical data
-2. Contextual data
```
Comment (jmandel) [inline spec/scopes-and-launch-context/index.md]: @Healthedata1 Why did you remove this list item? Scopes are still the way that an app requests contextual data (e.g. for the Standalone Launch flow).

**Comment by Healthedata1** [comment]
I changed to a numbered list instead of a bulleted list with numbers.  they should all be there.  
From...
-* 1. Clinical data
-* 2. Contextual data
-* 3. Identity data
To...
+1. Clinical data
+1. Contextual data
+1. Identity data
